styleguide-app/Debug/Control.elm

```diff
@@ -58,9 +58,13 @@ value initial =
 The first value will be the initial value.
 
 -}
-values : List a -> Control a
-values choices =
-    choice (List.map (\x -> ( toString x, value x )) choices)
+values : a -> List a -> Control a
```

Comment:
📓 We made this file pretty ugly in order to get it to compile, but also... not sure why we don't just use Nri.Select instead of this odd module.

Reply:
I'm not sure why his module is here.  It's from https://github.com/avh4/elm-debug-controls and looks like it was copied in here.  Maybe we can just remove it and go back to using the published package instead?

Reply:
Ah, sweet! Since that package isn't on 0.19 yet (I think?), using this one: https://package.elm-lang.org/packages/NoRedInk/elm-debug-controls-without-datepicker/latest/Debug-Control